streams/src/main/java/org/apache/kafka/streams/kstream/internals/MaterializedInternal.java

```diff
@@ -45,6 +45,18 @@ public MaterializedInternal(final Materialized<K, V, S> materialized,
         }
     }
 
+    public MaterializedInternal(final Materialized<K, V, S> materialized,
```

Comment:
Why do we need this not constructor?

Reply:
This is actually a residue due to refactoring, I've removed the constructor.